google/cloud/aiplatform/__init__.py

```diff
@@ -15,6 +15,4 @@
 # limitations under the License.
 #
 
-from google.cloud.aiplatform import gapic
```

Comment:
This was raising an "imported but unused error". Let me know if I should re-add it.